.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Editor specific files should be kept out of the repo :)

Reply:
While I would agree with you in the past, I think there are advantages in including settings in bigger project where many people work in the same editor and need to use the same tooling. But this is not a big project, so I'm removing them.